client/state/memberships/earnings/selectors.js

```diff
@@ -20,6 +20,6 @@ export function getEarningsWithDefaultsForSiteId( state, siteId ) {
 		last_month: earnings.last_month ?? 0,
 		forecast: earnings.forecast ?? 0,
 		currency: earnings.currency ?? 'USD',
-		commission: earnings.commission ?? '0.1',
+		commission: earnings.commission ?? false,
```

Comment:
Nitpick, but `null` seems like a better default value here than `false` insofar as the usual value is numeric, not boolean.

Reply:
Sure! 6daca27ed880dbb84331403778a16125cd4ecd30